deploy.yml

```diff
@@ -33,3 +32,5 @@
       - "{{ openshift_services }}"
       - "{{ openshift_jobs }}"
       - "{{ openshift_routes }}"
+
+    - import_tasks: tasks/create_route_aliases.yml
```

Comment:
Whe can use the [Ansible tags](https://docs.ansible.com/ansible/latest/user_guide/playbooks_tags.html) to deploy the app, the routes or th configs like this : 
```bash
docker run --rm -it \	
    --env-file env.d/production \	
    arnold \	
    ansible-playbook deploy.yml -e "customer=myCustomer env_type=myEnv" \
         --tags "configuration,app"
```
Or : 
```bash
docker run --rm -it \	
    --env-file env.d/production \	
    arnold \	
    ansible-playbook deploy.yml -e "customer=myCustomer env_type=myEnv" \
         --tags "routes"
```

It can be also on a next issues.

Reply:
Can you create a new issue to better describe what you have in mind?